crates/re_renderer/src/line_strip_builder.rs

```diff
@@ -135,23 +142,35 @@ where
         self
     }
 
+    /// Sets an outline mask for every element in the batch.
+    #[inline]
+    pub fn outline_mask(mut self, outline_mask: OutlineMaskPreference) -> Self {
```

Comment:
Nit: maybe name this to `outline_mask_id` to make it clear it's the id for the mask as opposed to the mask itself.